src/components/Book.js

```diff
@@ -13,19 +15,37 @@ function Book({ itemId, title, author }) {
 
   return (
     <>
-      <li>
+      <li className="Lesson-Panel">
         <div className="Book">
-          <input type="checkbox" />
-          {' '}
           <div className="tinga">
-            {title}
-            {'  '}
-            {author}
+            <p className="Action">{category}</p>
+            <h2 className="Kichwa">{title}</h2>
+            <p className="Author">{author}</p>
+          </div>
+          <div className="btns">
+            <button className="comments" type="button">Comments</button>
+            {/* <Button name="Remove" clickMe={deleteBook} /> */}
+            <button className="remove" type="button" onClick={deleteBook}>Remove</button>
+            <button className="edit" type="button">Edit</button>
+          </div>
+        </div>
+        <div className="Book-info">
+          <div className="Book-Percentage">
+            <div className="Progress-Percentage" />
+            <div className="Percentage-p">
+              <p className="Percentage-1">65%</p>
+              <p className="Percentage-2">Completed</p>
+            </div>
+          </div>
+          <div className="vertical-separator">
+            <p className="try">|</p>
+          </div>
+          <div className="Book-last">
+            <p className="current">CURRENT CHAPTER</p>
+            <p className="Chapter">Chapter 10</p>
+            <button className="Update" type="button">UPDATE PROGRESS</button>
           </div>
```

Comment:
- [ ] If you see the design then the navigation panel is at the top page covering the whole width. See the image below for reference.
### your design
![hhhhhhhhhh](https://user-images.githubusercontent.com/81727988/181936548-85175610-41e3-45c1-b0b3-c0c4db7ac35d.png)

### Design provided by Microverse
![figma](https://user-images.githubusercontent.com/81727988/181936485-2db6c1c8-5146-4c2e-a2ca-7e21675b5c13.png)